python-project-template/.github/workflows/testing-and-coverage.yml.jinja

```diff
@@ -36,3 +37,29 @@ jobs:
       uses: codecov/codecov-action@v5
       with:
         token: {% raw %}${{ secrets.CODECOV_TOKEN }}{% endraw %}
+{%- if test_lowest_version != 'none' %}
+  test-lowest-versions:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python {{ py.min(python_versions)  }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: '{{ py.min(python_versions)  }}'
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        python -m pip install --upgrade uv
+        uv venv venv
+        source venv/bin/activate
+        {%- if test_lowest_version == 'direct' %}
+        uv pip compile --resolution=lowest-direct -o requirements_lowest.txt pyproject.toml
+        {%- elif test_lowest_version == 'all' %}
+        uv pip compile --resolution=lowest -o requirements_lowest.txt pyproject.toml
```

Comment:
```suggestion
        uv pip compile --resolution=lowest-direct --only-binary=:all: -o requirements_lowest.txt pyproject.toml
        {%- elif test_lowest_version == 'all' %}
        uv pip compile --resolution=lowest --only-binary=:all: -o requirements_lowest.txt pyproject.toml
```

Reply:
I don't think we liked the results when this was added to the mix. My WIP PR to add this to HATS ultimately didn't use it: https://github.com/astronomy-commons/hats/pull/497